src/components/Item/styles/ItemHeader.module.css

```diff
@@ -0,0 +1,28 @@
+.itemWrap {
+    padding: var(--spacers-dp8);
+}
+
+.itemHeaderWrap {
+    display: flex;
+    align-items: top;
+    margin: var(--spacers-dp4) var(--spacers-dp4) var(--spacers-dp8)
+        var(--spacers-dp4);
```

Comment:
Could this be adjusted to: `margin: var(--spacers-dp4) var(--spacers-dp4) var(--spacers-dp8) var(--spacers-dp8);`

Reply:
Yepp.